connect/src/core.ts

```diff
@@ -393,6 +397,22 @@ export default class Ad4mConnect {
       this.setToken(null);
     }
 
+    if (this.isHosting) {
+      let token = localStorage.getItem('hosting_token');
+
+      const response = await fetch('https://hosting.ad4m.dev/api/service/checkStatus', {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + token
+        },
+      });
+
+      if (response.status === 200) {

```

Comment:
Should this be 
```suggestion
      if (response.status != 200) {

```
?